tests/staggered_invert_test.cpp

```diff
@@ -138,6 +138,13 @@ void init()
   // Set QUDA internal parameters
   gauge_param = newQudaGaugeParam();
   setStaggeredGaugeParam(gauge_param);
+  QudaGaugeSmearParam smear_param;
+  if (gauge_smear) {
+    smear_param = newQudaGaugeSmearParam();
+    setGaugeSmearParam(smear_param);
+  }
+
+  if (!inv_multigrid) setStaggeredInvertParam(inv_param);
```

Comment:
I think this can be removed?

Reply:
removed in 6925fbd407461ce9109a2238ff5caef85c84044c